### ISSUE TITLE
standby_cluster bootstrap using custom script without host and port

### ISSUE THREAD

**Issue by carolfly86** [zalando/patroni#999]
patroni version: 1.5.5

hi, I'm trying to bootstrap a standby_cluster with wal log shipping using custom script.
I would like the standby_cluster to not connect to the master, so I omitted the host and port info in the config, but the bootstrap would not  start without those info. 

``` 
2019-03-06 18:11:45,020 INFO: trying to bootstrap a new standby leader
 2019-03-06 18:11:45,021 ERROR: failed to bootstrap clone from remote master None
```

```
bootstrap:
  dcs:
    standby_cluster:
      create_replica_methods: ['wal_e']
      restore_command: "/usr/bin/envdir /etc/wal-e.d/env /opt/wal-e/bin/wal-e wal-fetch %f %p"
```

postgresql section contains the wal_e replica method:

```
postgresql:
  wal_e:
    command: "/var/opt/gitlab/patroni/scripts/wale-restore.sh"
```

**Comment by carolfly86** [comment]
I think this is fixed in this ? https://github.com/zalando/patroni/commit/e670122c805176a3689a7ddc8849a557038bde58

**Comment by CyberDem0n** [comment]
Hi,

in the `wal_e` section you need to specify `no_master: 1` and wale-restore should ignore `--no_master` argument.

You can also take a look how it is used in spilo:
https://github.com/zalando/spilo/blob/master/postgres-appliance/scripts/configure_spilo.py#L116-L132
https://github.com/zalando/spilo/blob/master/postgres-appliance/scripts/configure_spilo.py#L282-L289

**Comment by CyberDem0n** [comment]
#981 if fixing a different problem.
If there is more than one member in the standby cluster it makes possible to bootstrap them the same way as standby leader.

Actually there is one more fix coming in #998, but for you it might be not relevant.

**Comment by carolfly86** [comment]
thanks for your help! the `no_master: 1` attribute did work
